common4j/src/main/com/microsoft/identity/common/java/nativeauth/providers/responses/signin/SignInChallengeApiResponse.kt

```diff
@@ -39,7 +39,7 @@ import java.net.HttpURLConnection
  */
 data class SignInChallengeApiResponse(
     @Expose override var statusCode: Int,
-    @Expose @SerializedName("credential_token") val credentialToken: String?,
+    @Expose @SerializedName("continuation_token") val continuationToken: String?,
```

Comment:
Should not be annotated with @Expose
(not your change, but let's clean it up)

Reply:
Removed the annotation